src/vs/base/browser/dom.ts

```diff
@@ -1235,6 +1235,9 @@ export function asDomUri(uri: URI): URI {
 	if (Schemas.vscodeRemote === uri.scheme) {
 		return RemoteAuthorities.rewrite(uri);
 	}
+	if (platform.isNative && Schemas.file === uri.scheme) {
```

Comment:
@deepak1556 I somewhat like how we have a `RemoteAuthorities.rewrite(uri)` above that encapsulates the rewrite into a helper, should we do the same for this case here? E.g. introduce a `LocalFileAccess.rewrite(uri)` helper? If we did that, we could actually hide `Schemas.vscodeFileAuthority` as an implementation detail I think.